src/components/FilmStreaming.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
Tailwind css must be used for styling your component . Don't use style scoped css.